operators/src/plot/statistics.rs

```diff
@@ -173,10 +330,35 @@ where
     }
 }
 
+/// The statistics summary output type for vector input column
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct VectorColumnStatisticsOutput {
+    pub value_count: usize,
+    pub nan_count: usize,
+    pub min: f64,
+    pub max: f64,
+    pub mean: f64,
+    pub stddev: f64,
+}
```

Comment:
We thought of unifying the output for rasters and vectors.
Our suggestion:

```suggestion
struct StatisticsOutput {
    pub value_count: usize,
    pub valid_count: usize,
    pub min: f64,
    pub max: f64,
    pub mean: f64,
    pub stddev: f64,
}
```

So, we use (1) value for pixel for rasters and (2) valid_count instead of nan_count, which means, we have to calculate `number_statistics.count() - number_statistics.nan_count()`.

Reply:
Isn't `number_statistics.count()` already sort of a "valid_count", because it excludes NaN values?
Wouldn't it then make more sense to have "value_count" be `number_statistics.count() + number_statistics.nan_count()` and "valid_count" be `number_statistics.count()`?

Reply:
... like this https://github.com/geo-engine/geoengine/pull/614/commits/dcc29d1618711ec7ca0f13d42ec7056e7129b1a4

Reply:
You are right, didn't have the implementation in mind.